dapps/templates/boilerplate/package.json

```diff
@@ -21,7 +21,6 @@
     "clean": "npm run reset",
     "package": "npm pack",
     "qa": "npm-run-all test package",
-    "reset": "npx embark-reset && npx rimraf embark-*.tgz package",
```

Comment:
Please replace with a working equivalent:

```
"reset": "npx embark reset && npx rimraf embark-*.tgz package",
```

This script had a purpose.

Reply:
Just realized, my suggestion won't work very well. `yarn reboot[:full]` will bomb out if you don't have a build of `packages/embark` in place, i.e. so long as the reset code is inside `src/` and needs to be copied/transpiled to `dist/`. The `embark-reset` package doesn't get transpiled by babel, that was another aspect of its design, albeit that design decision was driven by having dapps inside the monorepo. Personally, I think dapps, including templates and tests, should be in a completely separate monorepo (a re-envisioned dapp-bin); embark commands in CI would be executed in that separate monorepo using a production build of embark.

Reply:
I think that for now, regardless of the "vision" I explained above, the cleanest thing for the reorg is to make `embark-reset` a package within `packages/core`, i.e. a simple move from `packages/embark-reset` to `packages/core/reset`.

That would allow the monorepo's `reboot[:full]` logic to function without hiccups and then the question of where the reset code properly belongs (hardwired into `packages/embark` or elsewhere) can be addressed later.